### ISSUE TITLE
No update "Discover new hosts last scan"

### ISSUE THREAD

**Issue by ElGranLoky** [phpipam/phpipam-agent#48]
Versión IPAM: 1.3.2
Versión Agent:  last version

I use the agent with:

/usr/local/bin/php /opt/phpipam-agent/index.php update
/usr/local/bin/php /opt/phpipam-agent/index.php discover

The two commands work correctly, but the discovery does not update the last scan field. Anyway in the description field of the ip if it shows when the equipment has been discovered as you can see in the image.

I do not know exactly where the problem is, any suggestions?

![image](https://user-images.githubusercontent.com/2712200/47413735-66bffd00-d76f-11e8-8aec-020a00e3e6ca.png)

**Comment by Haringstad** [comment]
@ElGranLoky I am facing exactly the same issue. 
When I run the agent, it perfectly scans the subnet, but no updates are performed.
Tried with PHP 5.6, 7.0, 7.1, 7.2, 7.3., but no dice......

**Comment by Haringstad** [comment]
YUP! Figured it out!!! My ISSUE was TZ! Because the system running the agent was in UTC, and the IP-DB master was in Europe/Amsterdam, nothing got ever updated. Created a script, calling the PHP indexes, with:
```
#!/bin/bash
export TZ=Europe/Amsterdam

for i in phpipam-agent-172 phpipam-agent-82
do
  php7.0 /opt/$i/index.php discover
  php7.0 /opt/$i/index.php update
done
```

**Comment by ElGranLoky** [comment]
I found one problem - #54
